server/proxy-server.js

```diff
@@ -110,10 +110,26 @@ function proxyGet (req, res) {
 
 server.get('*', proxyGet)
 
-server.listen(config.port, (err) => {
-  if (err) throw err
-  console.log(`Proxy Server running at port ${config.port}`)
-  console.log(`Acceptable origins: ${config.origins.split(';')}`)
-  console.log(`Acceptable targets: ${config.targets.split(';')}`)
-  console.log(`MS ML URL: ${config.url_for_msml}`)
-})
+if (NODE_ENV === 'production') {
```

Comment:
This clause fails when trying to run `npm run proxy-server`, and should be changed from `NODE_ENV` to `process.env.NODE_ENV`

```suggestion
if (process.env.NODE_ENV === 'production') {
```